autopush/tests/test_router.py

```diff
@@ -1015,18 +1028,18 @@ def test_route_to_busy_node_with_ttl_zero(self):
 
         def verify_deliver(fail):
             exc = fail.value
-            ok_(exc, RouterResponse)
-            eq_(exc.status_code, 201)
-            eq_(len(self.metrics.increment.mock_calls), 0)
-            ok_("Location" in exc.headers)
+            assert exc, RouterResponse
```

Comment:
I'm noticing a few of these asserts (originally ok_s) are incorrect. This and a couple others should be assert isinstance(exc, RouterResponse).  A couple "assert status.status_code, 201" below.

Logged a separate issue here:

https://github.com/mozilla-services/autopush/issues/1043

Reply:
Yeah, I had to correct a few of those. Good catch on these. I'll clean them up.